<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="generiqueBg">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="quitHyperyon">
          <property name="styleSheet">
           <string notr="true">background-color: white;
padding: 15px;
color: black;
border-radius: 20px;</string>
          </property>
          <property name="text">
           <string>quitte Hyperion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="launchHyperyon">
          <property name="styleSheet">
           <string notr="true">background-color: white;
padding: 15px;
color: black;
border-radius: 20px;</string>
          </property>
          <property name="text">
           <string>lance Hyperion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorLaunch">
          <property name="styleSheet">
           <string notr="true">background-color: white;
padding: 15px;
color: black;
border-radius: 20px;</string>
          </property>
          <property name="text">
           <string>envoi la couleur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelRed">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Rouge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderRed">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="valRed">
            <property name="styleSheet">
             <string notr="true">background-color: white;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelGreen">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Vert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderGreen">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="valGreen">
            <property name="styleSheet">
             <string notr="true">background-color: white;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="labelBlue">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Bleu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderBlue">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="valBlue">
            <property name="styleSheet">
             <string notr="true">background-color: white;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="connectionStatus">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>En attente de connexion   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHexa">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>code hexa :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editHexa">
          <property name="styleSheet">
           <string notr="true">color: white;
border: 0px;</string>
          </property>
          <property name="text">
           <string>#000000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPantone">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>code pantone :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPantone">
          <property name="styleSheet">
           <string notr="true">color: white;
border: 0px;</string>
          </property>
          <property name="text">
           <string>Black 6 C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucouleur">
    <property name="title">
     <string>couleur</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menucouleur"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
